.github/workflows/pip-build.yml

```diff
@@ -255,6 +267,35 @@ jobs:
       - name: Build
         run: msbuild -m  source\MeshLib.sln -p:Configuration=Release -p:PythonVersion=${{matrix.py-short-version}} -p:PythonLibPath=c:\hostedtoolcache\windows\python\${{steps.setup-python.outputs.python-version}}\x64\libs\${{matrix.py-file}}.lib -p:PythonDllPath=c:\hostedtoolcache\windows\python\${{steps.setup-python.outputs.python-version}}\x64\${{matrix.py-file}}.dll -p:PythonIncludePath=c:\hostedtoolcache\windows\python\${{steps.setup-python.outputs.python-version}}\x64\include\
 
+        # Install MSYS2 using their github action instead of ours `install_deps_windows_msys2.bat`. Local uses should use that bat file.
+        # There's no particular reason for not using our bat file, I'm just assuming the official action is somehow better.
+      - name: Install MSYS2 for MRBind
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: clang64
+          install: make mingw-w64-clang-x86_64-clang mingw-w64-clang-x86_64-clang-tools-extra mingw-w64-clang-x86_64-cmake
+          # Matching the location that `install_deps_windows_msys2.bat` uses, because that's what `install_mrbind_windows_msys2.bat` expects.
+          location: C:\msys64_meshlib_mrbind
+
+        # Move `C:\msys64_meshlib_mrbind\msys64\*` to the parent directory, that's what our .bat files expect.
+      - name: Adjust MSYS2 installation directory
+        shell: cmd
+        run: |
+          for %%a in ("C:\msys64_meshlib_mrbind\msys64\*") do move /y "%%~fa" C:\msys64_meshlib_mrbind
+          for /d %%a in ("C:\msys64_meshlib_mrbind\msys64\*") do move /y "%%~fa" C:\msys64_meshlib_mrbind
+          rmdir C:\msys64_meshlib_mrbind\msys64
```

Comment:
Why do our .bat files expect that?

Reply:
Removed this and added a setting to the bat files.